packages/monaco/src/browser/monaco-editor-provider.ts

```diff
@@ -287,6 +287,7 @@ export class MonacoEditorProvider {
         const options = this.createOptions(this.preferencePrefixes, model.uri, model.languageId);
         options.model = model.textEditorModel;
         options.readOnly = model.readOnly;
+        options.lineNumbersMinChars = 3;
```

Comment:
As an alternative to the current way:
The value could be placed somewhere as constant, [here](https://github.com/eclipse-theia/theia/blob/bb17f711ffcf0cee12f24e95540a56b236c006a5/packages/monaco/src/browser/monaco-editor-model.ts#L49), for example.
Then 
```
options.lineNumbersMinChars = model.lineNumbersMinChars;
```
wdyt?